ios/RCTMapboxGL/RCTMapboxGLManager.m

```diff
@@ -43,6 +43,20 @@ - (NSArray *)customDirectEventTypes
     ];
 }
 
+- (NSDictionary *)constantsToExport
+{
+    return @{
+             @"styles": @{
+                 @"light": [[MGLStyle lightStyleURL] absoluteString],
+                 @"dark": [[MGLStyle darkStyleURL] absoluteString],
+                 @"streets": [[MGLStyle streetsStyleURL] absoluteString],
+                 @"emerald": [[MGLStyle emeraldStyleURL] absoluteString],
+                 @"satellite": [[MGLStyle satelliteStyleURL] absoluteString],
+                 @"satelliteHybrid": [[MGLStyle hybridStyleURL] absoluteString],
```

Comment:
We were planning to call this style just Hybrid in iOS SDK documentation, since that’s what Google and Apple call it.